test/spec/testers/rule-tester-spec.js

```diff
@@ -0,0 +1,31 @@
+import { getTitle } from '../../../lib/testers/rule-tester';
+
+import { expect } from '../../helper';
+
+
+describe('rule-tester', function() {
```

Comment:
Adding the bare minimum in terms of test coverage. I love your efficiency! :heart:

Reply:
I'd prefer an actual usage test that verifies the thing does not blow up.

Such as adding a name or two to the existing tests.

Reply:
Cf. https://github.com/bpmn-io/bpmnlint/blob/master/test/rules/end-event-required.js#L1.

Reply:
Will add.